pkg/cmdlets/git/git.go

```diff
@@ -115,12 +115,12 @@ func ensureCorrectProject(owner, name, projectFilePath string, out output.Output
 	}
 
 	if !(strings.ToLower(proj.Owner()) == strings.ToLower(owner)) || !(strings.ToLower(proj.Name()) == strings.ToLower(name)) {
-		out.Print(locale.T("warning_git_project_mismatch"))
+		out.Notice(locale.Tr("warning_git_project_mismatch", repoURL, project.NewNamespace(owner, name, "").String(), constants.DocumentationURL))
```

Comment:
`constants.DocumentationURL` is the top level docs URL. This should be the specific docs page for the git project missmatch.